zeppelin-integration/src/test/java/org/apache/zeppelin/WebDriverManager.java

```diff
@@ -80,83 +80,44 @@ public WebDriver getWebDriver() {
   }
 
   private WebDriver constructWebDriver(int port) {
-    Supplier<WebDriver> chromeDriverSupplier = () -> {
-      try {
-        ChromeOptions options = new ChromeOptions();
-        return new ChromeDriver(options);
-      } catch (Exception e) {
-        LOG.error("Exception in WebDriverManager while ChromeDriver ", e);
-        return null;
-      }
-    };
-    Supplier<WebDriver> firefoxDriverSupplier = () -> {
-      try {
-        return getFirefoxDriver();
-      } catch (Exception e) {
-        LOG.error("Exception in WebDriverManager while FireFox Driver ", e);
-        return null;
-      }
-    };
-    Supplier<WebDriver> safariDriverSupplier = () -> {
-      try {
-        return new SafariDriver();
-      } catch (Exception e) {
-        LOG.error("Exception in WebDriverManager while SafariDriver ", e);
-        return null;
-      }
-    };
+    WebDriver driver = getFirefoxDriver();
 
-    WebDriver driver;
-    switch (SystemUtils.getEnvironmentVariable("ZEPPELIN_SELENIUM_BROWSER", "").toLowerCase(Locale.ROOT)) {
```

Comment:
I think this switch with the environment variable is quite good; we should use this behavior in the tests to start a Firefox browser.